plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -40,7 +42,7 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
     ProgramConversionContext {
     private val methods: MutableMap<MangledName, FunctionEmbedding> = SpecialKotlinFunctions.byName.toMutableMap()
     private val classes: MutableMap<MangledName, ClassTypeEmbedding> = mutableMapOf()
-    private val properties: MutableMap<MangledName, PropertyEmbedding> = mutableMapOf()
+    private val properties: MutableMap<CallableId, PropertyEmbedding> = mutableMapOf()
```

Comment:
Why the change here?  Our mangled names should already be able to uniquely represent callable ids, but I could imagine we may want to have names that are synthetic, too.